editions/pluginlibrary/tiddlers/GettingStarted.tid

```diff
@@ -0,0 +1,11 @@
+title: GettingStarted
+
+This server let's you test the plugin import mechanism. 
+
+!! Important
+
+''Content isn't saved back to the server. It's read only!''
```

Comment:
Please avoid exclamation marks. They should not be used in technical documentation.

Also we don't say "isn't", we spell it out in full as "is not".

Clearer to say:

The default setup is read-only. Nothing is saved back to the server.